argus/introspection/cloud/windows.py

```diff
@@ -194,6 +193,44 @@ def get_cbinit_key(execute_function):
     return key_x64
 
 
+def get_os_version(client, field):
+    cmd = "[System.Environment]::OSVersion.Version.{}".format(field)
+    result, _, _ = client.run_command_with_retry(cmd, command_type=util.POWERSHELL)
+    return util.get_int_from_str(result.strip())
+
+
+def parse_netsh_output(output):
```

Comment:
all these netsh commands should be put in one command: parse_netsh_output, which returns a formatted dictionary, like

@{"mtu"=1500;"name"=""}...